src/Microsoft.Framework.ConfigurationModel.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.ConfigurationModel.UserSecrets;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
+
+namespace Microsoft.Framework.ConfigurationModel
+{
+    public static class ConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds the user secrets configuration source.
+        /// </summary>
+        /// <param name="configuration"></param>
+        /// <returns></returns>
+        public static IConfigurationSourceRoot AddUserSecrets([NotNull]this IConfigurationSourceRoot configuration)
```

Comment:
Another overload where the appSecretId is passed in directly might be nice, seems more in-line with all our other apis.